tsconfig.json

```diff
@@ -41,7 +41,7 @@
     // "baseUrl": "./",                       /* Base directory to resolve non-absolute module names. */
     // "paths": {},                           /* A series of entries which re-map imports to lookup locations relative to the 'baseUrl'. */
     // "rootDirs": [],                        /* List of root folders whose combined content represents the structure of the project at runtime. */
-    // "typeRoots": [],                       /* List of folders to include type definitions from. */
+    "typeRoots": ["/"],                       /* List of folders to include type definitions from. */
```

Comment:
Why do we need this?

Reply:
It tells tsnode to look for the types.d.ts file which extends the Request interface.

a TSError is thrown otherwise

Reply:
This is weird, TS should pick that file up automatically.

Reply:
Yeah I was confused too but this option seems to be the solution